### ISSUE TITLE
[Bug]: Great rune option missing, even after getting and activating multiple great runes.

### ISSUE THREAD

**Issue by Redie96** [LukeYui/EldenRingSeamlessCoopRelease#127]
### Discord Username

ikeThun#1929

### Describe the issue in detail

So I do not know which part could be useful, so I'll try to explain everything that has happened.

A friend and I started the seemless coop on patch 1.0.1, then after killing Margit, a 3rd friend joined us. Fast forward to our Godrick kill, we all get the great runes, all cool so far. Then one of the 2 guys leaves, and we 2 go and activate the great rune atop the divine tower. Happily, we fast travel back to Roundtable Hold, only to notice, there is no Enia (Note: I was hosting). We used separation mist, and there was still no Enia, so i decided to go to the divine tower again, to see if i could activate the rune again. The answer was no, but after a quick reboot of the game i could. Now Enia also showed up, but only shows up even now on the latest patch, if I'm alone in a game, or someone else is hosting. She never shows up when I'm hosting and there are people joined.

There are now multiple great runes sitting in my inventory, and i am afraid to activate them, because they will just disappear.

The next issue arose quickly, i had no great runes options when sitting at a grace. It's just not there.

tl;dr: Got great rune, activated it, disappeared from my inventory, can't use it.

PS: On version i selected 1.0.1, because that's when it started, but we all keep our addons up-to-date

### Steps to recreate the bug

- Host game
- Kill Godrick
- Activate Great Rune
- While still hosting go to roundtable hold
- Use separation mist
- Activate great rune again if you can
- try to use a great rune

### Version

1.0.1

**Comment by Amnesiac9** [comment]
Similar issue, but the option is there for me after I activated the great rune after we killed godrick, but my buddy lost his great rune and does not have the option.
